include/seahorn/Expr/Smt/Z3.hh

```diff
@@ -612,9 +611,17 @@ public:
       *(out++) = z3.toExpr(r[i]);
   }
 
+  std::string getReasonUnknown() {
+    return std::string(Z3_solver_get_reason_unknown(ctx, solver));
+  }
+
   boost::tribool solve() {
     boost::tribool res = z3l_to_tribool(Z3_solver_check(ctx, solver));
     ctx.check_error();
+    if (indeterminate(res)) {
+      llvm::errs() << "Reason (for unknown status) : " << getReasonUnknown()
```

Comment:
should be under `LOG`. Probably don't want to print this from inside solve. This is client dependent.

Reply:
is this being addressed and PR merged?